### ISSUE TITLE
Error on message extraction

### ISSUE THREAD

**Issue by igorline** [drd/jsxlate#1]
Hi! I'm receiving error on message extraction.
Was downloading zip from the page and trying to extract from examples/simple.
This is log that I am having

```
Error in file examples/simple/app.jsx:
Error: Unexpected input: function Function() { [native code] } undefined {"type":"Identifier","start":35,"end":39,"loc":{"start":{"line":2,"column":8},"end":{"line":2,"column":12}},"name":"I18N"}
```

**Comment by drd** [comment]
Hi Igor, can you let me know what version of node & jsxlate you're using, and the exact command you used to extract the messages? When I run

```
bin/extract-messages.js examples/simple/app.jsx
```

Using version 0.2.11, I do not see any errors.

Thanks!

**Comment by drd** [comment]
Ah, from a fresh `npm install` I am able to replicate. Investigating…

**Comment by drd** [comment]
The issue was caused by Babel's (new!) Babylon parser adding a key to the Node prototype in babel/babel#1978. Fixed in `0.2.12`.
